src/domain/Spacers/Margin/Margin.tsx

```diff
@@ -1,19 +1,27 @@
 import React from 'react'
+import { FlattenSimpleInterpolation } from 'styled-components'
 
 import { Props } from '../../../common'
+import { styleForMargins } from '../services/margins'
 import { StyledMargin } from './style'
 
-const Margin: React.FC<{
+type IMargin = React.FC<{
   margins?: Props.IMargins
   children?: React.ReactNode
-}> = ({margins, children}) =>  (
+}> & {
+  styleForMargins: (margins?: Props.IMargins) => FlattenSimpleInterpolation | null
```

Comment:
can use `typeof styleForMargins` here I think to save yourself some hassle if that changes